evap/staff/importers.py

```diff
@@ -710,11 +713,13 @@ def save_users_to_db(self):
             msg = _("No users were created.")
         else:
             msg = format_html(
-                _("Successfully created {object}: {list}"),
-                object=ngettext("{user_count} user", "{user_count} users", len(created_users)).format(
-                    user_count=len(created_users)
-                ),
-                list=create_user_list_html_string_for_message(created_users),
+                _("{}: {}"),
```

Comment:
If we were really serious about internationalization with the goal of supporting all languages, we would have to do something to allow translating this here. With german and english, however, I think we don't have to translate this one.